code/go/0chain.net/smartcontract/dbs/event/user.go

```diff
@@ -53,17 +54,27 @@ func (edb *EventDb) addOrOverwriteUser(u User) error {
 	return result.Error
 }
 
+func (edb *EventDb) GetUserFromId(userId string) (User, error) {
+	user := User{}
+	return user, edb.Store.Get().Model(&User{}).Where(User{UserID: userId}).Scan(&user).Error
+
+}
+
+func (edb *EventDb) CreateUser(usr *User) error {
```

Comment:
is this being used anywhere?

Reply:
I don't remember adding this, could be a result of bad merge/rebase. Removed it since couldn't find it being used anywhere.